includes/Generator/Generator.php

```diff
@@ -68,30 +69,57 @@ protected static function generate_term_ids( $limit, $taxonomy ) {
 			return $term_ids;
 		}
 
-		$terms = self::$faker->words( $limit );
+		$words       = str_word_count( $name, 1 );
+		$extra_terms = str_word_count( implode( ' ', self::$faker->department( $limit ) ), 1 );
```

Comment:
@ovidiul `self::$faker->department( $limit )` returns a string vs an array causing the `implode` to throw a warning.

Reply:
fixed, my bad!